optimum/intel/ipex/modeling_base.py

```diff
@@ -146,7 +146,8 @@ def __init__(
         self.use_cache = kwargs.get("use_cache", False)
         self.model_save_dir = model_save_dir
         self._add_patch = _is_patched_with_ipex(model, self.export_feature, self.use_cache)
-        self.compiled = False
+        self.compile = self.can_compile()
+        model.config.compile = self.compile
```

Comment:
same thing, "self.compile" don't need to be public.

Reply:
fixed